pom.xml

```diff
@@ -216,6 +217,62 @@
     </build>
 
     <profiles>
+		<profile>
+			<id>rpm</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.codehaus.mojo</groupId>
+						<artifactId>rpm-maven-plugin</artifactId>
+						<version>2.1.5</version>
+						<executions>
+							<execution>
+								<id>generate-rpm</id>
+								<goals>
+									<goal>rpm</goal>
+								</goals>
+							</execution>
+						</executions>
+						<configuration>
```

Comment:
* I think we've lost the package description.
* Does the license info get pulled from elsewhere in the pom automatically?
* I assume version info is picked up automatically from elsewhere in the pom too?